api/services/control/control.proto

```diff
@@ -206,6 +206,7 @@ message BuildHistoryRecord {
 	bool pinned = 14;
 	int32 numCachedSteps = 15;
 	int32 numTotalSteps = 16;
+	int32 numCompletedSteps = 17;
```

Comment:
I guess completed steps also include `DONE` steps or is it unrelated?

Reply:
Yes, that is is